editor/editor_node.cpp

```diff
@@ -5575,6 +5583,9 @@ void EditorNode::_feature_profile_changed() {
 }
 
 void EditorNode::_bind_methods() {
+	GLOBAL_DEF("editor/scene/scene_naming", SCENE_NAME_CASING_SNAKE_CASE);
```

Comment:
Should it be kept defaulting to "Auto" for 3.x?
Unlike 4.0 where users are kind of expected to start projects from scratch, this will throw off many 3.4 users who will see their naming scheme change unexpectedly (many tend to just use the suggested PascalCase name currently).

Reply:
I updated this PR to use "Auto" by default.